tests/components/evil_genius_labs/test_light.py

```diff
@@ -0,0 +1,76 @@
+"""Test Evil Genius Labs light."""
+from unittest.mock import patch
+
+import pytest
+
+
+@pytest.mark.parametrize("platforms", [("light",)])
```

Comment:
Single parameters don't need tuples. A list is enough.

Reply:
We're patching a value that is a tuple.